core/src/main/kotlin/com/imashnake/animite/core/extensions/MarkdownStyleExt.kt

```diff
@@ -0,0 +1,42 @@
+package com.imashnake.animite.core.extensions
+
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.RectangleShape
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.boswelja.markdown.style.BlockQuoteStyle
+import com.boswelja.markdown.style.CodeBlockStyle
+import com.imashnake.animite.core.R
+
+@Composable
+fun animiteBlockQuoteStyle(
+    background: Color = MaterialTheme.colorScheme.surfaceVariant,
+    barWidth: Dp = dimensionResource(R.dimen.markdown_block_corner_radius),
+    barColor: Color = MaterialTheme.colorScheme.onSurfaceVariant,
+    barShape: Shape = RectangleShape,
+    innerPadding: PaddingValues = PaddingValues(8.dp)
+) = BlockQuoteStyle(
+    background = background,
+    shape = RoundedCornerShape(barWidth),
```

Comment:
Not gonna define this as part of the theme?

Reply:
[Shape.kt](https://github.com/imashnake0/Animite/blob/main/app/src/main/java/com/imashnake/animite/features/theme/Shape.kt) is currently empty, I'm in the process of creating components in Figma so we'll have a more solid design system soon. I'll fix typography, shapes, colors, and themes in a follow up PR.